cluster-provision/gocli/cmd/run.go

```diff
@@ -432,7 +432,7 @@ func run(cmd *cobra.Command, args []string) (retErr error) {
 	wg := sync.WaitGroup{}
 	wg.Add(int(nodes))
 	// start one vm after each other
-	macCounter := 0
+	macCounter := 1
```

Comment:
This is not strictly necessary (anymore), since I don't set `--dhcp-host` for nodes on secondary networks, since they don't need an IP on those.
But I left it there for consistency:
On the primary network, this number is the node number. Without my change, on secondary networks, this number is the node number - 1.